modules/parquet/src/parquetjs/compression.ts

```diff
@@ -35,7 +35,7 @@ function toArrayBuffer(buffer: Buffer): ArrayBuffer {
 // import brotliDecompress from 'brotli/decompress';
 import lz4js from 'lz4js';
 // import lzo from 'lzo';
-// import {ZstdCodec} from 'zstd-codec';
+import {ZstdCodec} from 'zstd-codec';
```

Comment:
The reason that this was not included by default is that the library is enormous, like 2.5MB just for that compression. 

I made it so that applications can choose to import zstd themselves, or just use the compression worker which includes it.

You can try to check size of the latest version of the module see if it is smaller.